configuration/pih/scripts/visit/encounterTypeConfig.js

```diff
@@ -2011,20 +2011,37 @@ angular.module("encounterTypeConfig", [])
 
         // obgyn
         encounterTypes["d83e98fd-dc7b-420f-aa3f-36f648b4483d"] = {
-            DEFAULT: {
-                defaultState: "short",
-                shortTemplate: "templates/encounters/defaultEncounterShort.page",
-                longTemplate: "templates/encounters/defaultEncounterShort.page",
-                icon: "fas fa-fw fa-female",
-                editUrl: hfeStandardEditUrl,
-                showOnVisitList: true,
-                sections: [
-                    obgynInitial,
-                    ancVaccinations,
-                    primaryCareExam,
-                    obgynDx,
-                    obgynPlan
-                ]
+            versions: {
+                DEFAULT: {
+                    defaultState: "short",
+                    shortTemplate: "templates/encounters/defaultEncounterShort.page",
+                    longTemplate: "templates/encounters/defaultEncounterShort.page",
+                    icon: "fas fa-fw fa-female",
+                    editUrl: hfeStandardEditUrl,
+                    showOnVisitList: true,
+                    sections: [
+                        obgynInitial,
+                        ancVaccinations,
+                        primaryCareExam,
+                        obgynDx,
+                        obgynPlan
+                    ]
+                },
+                1.0: {
```

Comment:
Oh, annoying, I forgot that we *did* end up having to add support to modifying the encounterTypeConfig to get form versions to work properly?  it's possible you only need to do this if you are changing the sections that appear on a form in a new version?

Reply:
Hmm.  Yeah, it's possible that this change wasn't actually needed in the end.